### PR TITLE
Update year in LICENSE.txt

### PR DESCRIPTION
**Pull request by djhoese**
- [ ] Closes #xxxx
- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks David! 😄

Should we do the same thing in [distributed]( https://github.com/dask/distributed )?

**Comment by martindurant** [comment]
We agreed elsewhere to simply remove the latter date, rather than to have to keep updating it.

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2014-2018, Anaconda, Inc. and contributors
+Copyright (c) 2014-2022, Anaconda, Inc. and contributors
```
Comment (jrbourbeau) [inline LICENSE.txt:3]:
```suggestion
Copyright (c) 2014, Anaconda, Inc. and contributors
```

**Review by jrbourbeau — COMMENTED** [review]
Thanks @djhoese

**Comment by djhoese** [comment]
Just curious, is there agreement between the legal folks that having a single year in a copyright covers "to present"? How would there ever be an expired copyright?

**Comment by martindurant** [comment]
> Just curious, is there agreement between the legal folks that having a single year in a copyright covers "to present"?

It seems yes. Does copyright assigned to a company expire ever?

**Comment by jakirkham** [comment]
Some interesting perspectives:

https://softwareengineering.stackexchange.com/q/130478
https://softwareengineering.stackexchange.com/q/210472
https://softwareengineering.stackexchange.com/q/227812

**Comment by jakirkham** [comment]
> We agreed elsewhere to simply remove the latter date, rather than to have to keep updating it.

Interesting. Think I missed (or perhaps forgot) this conversation. Do you have a reference?

> Does copyright assigned to a company expire ever?

IANAL, but I believe so. In [the US think this is 95yrs]( https://www.copyright.gov/help/faq/faq-duration.html ).

In any event if we are curious about this, maybe it makes sense to just ask NumFOCUS.

**Comment by jakirkham** [comment]
> In any event if we are curious about this, maybe it makes sense to just ask NumFOCUS.

Brought this up on the NF Slack channel and was pointed to this ( https://stackoverflow.com/q/2390230 ) by another dev.

**Comment by jsignell** [comment]
I'm going to merge this since having an implied to present is clearly better than having an out of date end date :)

**Pull request merged**